pkg/objstore/azure/azure.go

```diff
@@ -211,7 +211,7 @@ func (b *Bucket) Exists(ctx context.Context, name string) (bool, error) {
 }
 
 // Upload the contents of the reader as an object into the bucket.
-func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader) error {
+func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader, sizeBytes int64) error {
```

Comment:
@vglafirov is this something we can use in azure?

If not @adrien-f let's make it `_` as well